FWCore/Services/src/SiteLocalConfigService.cc

```diff
@@ -26,9 +26,36 @@ using namespace xercesc;
 //<<<<<< MEMBER FUNCTION DEFINITIONS                                    >>>>>>
 
 namespace {
+
+  template< class CharType >
```

Comment:
I just checked the FWLite page and I see lots of FWCore stuff that should not be in FWLite. E.g. FWCore/Framework, FWCore/ServiceRegistry, FWCore/PythonUtilities, FWCore/PluginManager, and FWCore/PythonParameterSet. Where did those dependencies come from?
So from this FWLite should not depend on FWCore/Concurrency and it shouldn't depend on Xerces.